package.json

```diff
@@ -21,7 +21,7 @@
     "build:js-dev": "webpack --mode development",
     "build:js": "npm run lint & npm run format & webpack --mode production",
     "build:py": "node ./extract-meta src/lib/components > dash_bio/metadata.json && python get_version_info.py && python -c \"import dash; dash.development.component_loader.generate_classes('dash_bio', 'dash_bio/metadata.json')\"",
-    "build:all": "npm run build:js & npm run build:py",
+    "build:all": "npm run build:js && npm run build:py",
```

Comment:
`build:js`, `test`, and `build:all-dev` also have `&` - seems like all of these want `&&`? unless `&` does some magic with exit states...